pkg/processor/trigger/cron/trigger.go

```diff
@@ -80,7 +81,17 @@ func newTrigger(logger logger.Logger,
 	} else if configuration.Schedule != "" {
 		newTrigger.tickMethod = tickMethodSchedule
 
-		newTrigger.schedule, err = cronlib.Parse(configuration.Schedule)
+		// make sure the given schedule structure contains all 6 fields (Second, Minute, Hour, ...)
+		splitSchedule := strings.Split(configuration.Schedule, " ")
+		if len(splitSchedule) != 6 {
+			return nil, errors.Errorf("Cron trigger schedule doesn't have all needed 6 fields: %+v", configuration.Schedule)
+		}
+
+		// change the seconds to be automatically 0 (to prevent the method from running every second)
+		splitSchedule[0] = "0"
```

Comment:
Shouldn't we add a minimum sleep time between events of `time.Second`?

Reply:
There's no need to sleep. when setting seconds to be 0 - it will prevent it from running more than once in the same second (which was the reason we thought to add 1 second sleep for)